### ISSUE TITLE
"A Day in the Life of I2P"

### ISSUE THREAD

**Issue by anonimal** [monero-project/kovri#70]
Cross-posting with https://trac.i2p2.de/ticket/1747
> A long term project I imagine. Something that wouldn't be done anytime soon, but very nice to have. Below is a public excerpt from #i2p-dev. It was a continuation from a previous discussion with z3r0fox:
```
2015-12-23 05:19:22     +anonimal       z3r0fox: Re: illustrations, in summary; I was thinking of "A Day in the Life of I2P": a single graphic illustration that covers every layer and protocol used (or available for use) between two points.
2015-12-23 05:19:39     +anonimal       Something that combines the few existing illustrations available with a new one to form a cohesive, single point of reference for new and exisiting devs.
2015-12-23 05:20:11     +anonimal       From router start, information exchange, to shutdown.
2015-12-23 05:20:31     +anonimal       z3r0fox: I think we can fit it into one graphic. What do you think?
2015-12-23 13:22:56     +anonimal       z3r0fox: I'd like to start on sketching it out after the new year. I think I'll work on some ASCII art, lol
2015-12-23 13:26:44     z3r0fox anonimal: Whoa. https://alteeve.ca/w/AN!Cluster_Tutorial_2#A_Map.21 (ASCII art goals)
```
> Once I get a better grasp of *everything*, I can start on some ASCII art. z3r0fox said he has experience with graphic design. Perhaps we could combine our resources to produce a unified graphic.

This would be a benefit to both present and future Kovri developers.

**Comment by majestrate** [comment]
Would that be a graphical representation of the i2p protocol stack?

**Comment by anonimal** [comment]
Yes, I see that as being a portion of the illustration but not the entire illustration as a whole.

I remember seeing a Tor graphic quite some time ago, and I cannot for the life of me seem to find it anywhere. It's not a part of their official documentation so it was floating around somewhere. The closest remotely related thing that I could quickly find was https://www.eff.org/pages/tor-and-https.

Essentially, our graphic could illustrate a sense of timeline; a 24 hour period that includes floodfill rotation and real-time events. Possibly a combination of the EFF graphic(s) and https://alteeve.ca/w/AN!Cluster_Tutorial_2#A_Map.21 but with timestamps and scenarios.

This may all be too big for one graphic, we'll see. I wonder what we can come up with.
